Testscripts/Linux/fpga-card-validation.sh

```diff
@@ -123,8 +123,35 @@ function validate_cards() {
         SetTestStateAborted
         return 1
     fi
+    echo "****************************" >> TestExecution.log
+    echo "Running xbutil scan" >> TestExecution.log
+    echo "****************************" >> TestExecution.log
     xbutil scan >> TestExecution.log
+
+    echo "****************************" >> TestExecution.log
+    echo "Running xbutil validate" >> TestExecution.log
+    echo "****************************" >> TestExecution.log
+    xbutil validate >> TestExecution.log
```

Comment:
xbutil validate >> TestExecution.log 2>&1
update redirect like this way, then error and info log will be written into TestExecution.log.

Reply:
Done